modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_aggregation/angle_aggregation.h

```diff
@@ -26,38 +26,43 @@ namespace lbs
  * The octant based separation is achieved via the notion of "AngleSetGroup"
  * which will group angle sets for each quadrant or octant
  * (depending on 2D or 3D).*/
+
```

Comment:
Could you move the closing `*/` on its own line and have the doxygen comment attached to the class declaration, i.e. no empty line like this?

Reply:
Fixed.